packages/scss/src/components/_table.scss

```diff
@@ -621,4 +621,47 @@ $max-col-width: 20; //value in rem
 			right: $i * 1rem;
 		}
 	}
+}
+
+//columns sticky css drop shadow on scroll
+.tableColumnStickyWrapper {
+	$columnStickyShadow-backgroundSize-width: 10px;
+	overflow: auto;
+	max-width: 100%;
+	background-image: 
+		/* Shadows */ 
+		linear-gradient(to right, _color(white), _color(white)),
+		linear-gradient(to right, _color(white), _color(white)),
+		/* Shadow covers */ 
+		linear-gradient(to right, rgba(0,0,0,.25), rgba(255,255,255,0)),
+		linear-gradient(to left, rgba(0,0,0,.25), rgba(255,255,255,0));   
+
+	background-position:
+		var(--columnStickyShadow-leftOffset) center, 
```

Comment:
Don't use css-vars directly. prefer using our theming system. It will be able to switch from hard values to css-vars easily with a single bool.